processor/src/services/adyen-payment.service.ts

```diff
@@ -400,7 +400,11 @@ export class AdyenPaymentService extends AbstractPaymentService {
       if (e instanceof UnsupportedNotificationError) {
         log.info('Unsupported notification received', { notification: JSON.stringify(opts.data) });
         return;
+      } else if (e instanceof Errorx && e.httpErrorStatus === 404) {
+        log.info('Payment not found hence accepting the notification', { notification: JSON.stringify(opts.data) });
+        return;
```

Comment:
> However if the server is down or something similar then we could also get a 404 thus we will also accept the notification. While the actual payment id could still exist.

Not sure if I got this. That if is in a place where basically we are checking the errors returned by CoCo when updating a payment. If CoCo is down, we are not going to get a 404. Also, take into account that we are wrapping the known CoCo errors in a class that extends Errorx so if we get a not handled error, it won't be instance of Errorx. 

Certainly, we could improve this by returning specific errors from the paymentsSDK (ie. ResourceNotFound) but it would only help to improve the readability of the code since the way to know that this is a ResourceNotFound error is because we get a 404 from CoCo. I didn't do that now in order to avoid modifying the payments sdk and delaying the fix.